src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -133,20 +392,58 @@
 		"sourcePath" : "log.sagebase.org/teamSnapshots/",
 		"tableDescriptor": {
 			"name": "teamsnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"snapshot_timestamp": "timestamp",
-				"id": "bigint",
-				"name": "string",
-				"can_public_join": "boolean",
-				"created_on": "timestamp",
-				"created_by": "bigint",
-				"modified_on": "timestamp",
-				"modified_by": "bigint"
-			},
-			"partitionKeys": {
-				"snapshot_date": "date"
+			"description" : "This table contain snapshots of teams. Snapshots are taken when teams are created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
```

Comment:
Same question as above for distinguishing the difference between snapshots created via node creation/update/deletion vs the periodic snapshot taking.

Reply:
Same explanation as above